app/src/main/res/layout/admin_all_list_event_item.xml

```diff
@@ -10,13 +10,13 @@
     android:layout_marginStart="14dp">
 
     <com.google.android.material.imageview.ShapeableImageView
-        android:id="@+id/o_entrant_image_view"
+        android:id="@+id/a_entrant_image_view"
         android:layout_width="140dp"
         android:layout_height="140dp"
         android:layout_marginEnd="25dp"
         android:scaleType="centerCrop"
         app:shapeAppearanceOverlay="@style/RoundedSquare"
-        android:src="@drawable/moo_deng">
+        android:src="@drawable/default_poster">
```

Comment:
just wondering, what is this for?

Reply:
this is the default profile picture icon before the database loads new ones

Reply:
this shouldnt matter right? since a random image is generated anyways

Reply:
yuh, but the empty pic is better than moo deng